### ISSUE TITLE
Tomcat user has no home; setting TOMCAT_HOME has no effect

### ISSUE THREAD

**Issue by toKrause** [bitnami/bitnami-docker-tomcat#86]
**Description**

The user that runs Tomcat inside the container has no meaningful home folder. Setting the `TOMCAT_HOME` environment variable as described in the readme file has no discernible effect. Additionally, a Java web application that is deployed to Tomcat can't retrieve the users home folder. Running `System.getProperty("user.home")` returns `?`.

**Steps to reproduce the issue:**

*Default configuration:*

1. Create a `docker-compose.yml` with the following content:
```yaml
version: '2'

services:
  tomcat:
    image: bitnami/tomcat:8.5
    ports:
      - 8080:8080
```

2. Run Docker Compose
```bash
docker-compose up -d
```

3. Bash into the container
```bash
docker exec -it "$(docker-compose ps tomcat | sed -n '3p' | awk '{ print $1 }')" bash
```

4. Inspect home folder
```bash
echo $HOME
````

*Modified configuration:*

1. Create a `docker-compose.yml` with the following content:
```yaml
version: '2'

services:
  tomcat:
    environment:
      - TOMCAT_HOME=/home/foobar
    image: bitnami/tomcat:8.5
    ports:
      - 8080:8080
    volumes:
      - ./home:/home/foobar
```

2. Create a local folder with correct permission
```bash
mkdir home
sudo chown 1001:1001 home
```

3. Run Docker Compose
```bash
docker-compose up -d
```

4. Bash into the container
```bash
docker exec -it "$(docker-compose ps tomcat | sed -n '3p' | awk '{ print $1 }')" bash
```

5. Inspect home folder
```bash
echo $HOME
````

**Describe the results you received:**

The home folder of the user that runs Tomcat is is not configures properly. `$HOME` points to the root folder. A folder that is  not writable by the user and therefore not usable as a home folder. The alleged default home folder `/home/tomcat` doesn't even exist. The system file `/etc/passwd` has no entry for the user. Setting the `TOMCAT_HOME` environment variable has no effect. 

**Describe the results you expected:**

The alleged default home folder `/home/tomcat` of the user that runs Tomcat should exist and be configured as that user home folder. Setting the `TOMCAT_HOME` environment variable should change that configuration.

**Additional information you deem important (e.g. issue happens only occasionally):**

I created and deployed a small Java web application that provides the following Servlet:

```java
@WebServlet("/folders")
public class ExampleServlet extends HttpServlet {

    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {

        response.setContentType("text/plain");

        PrintWriter out = response.getWriter();
        out.println("HOME:              " + System.getenv("HOME"));
        out.println("Home Folder:       " + System.getProperty("user.home"));
        out.println("Working directory: " + System.getProperty("user.dir"));

    }

}
```

Opening the corresponding URL yields the following result:

```
HOME:              /
Home Folder:       ?
Working directory: /
```

Since ` System.getProperty("user.home")` is the usual way to retrieve the user's home folder in a Java application (in fact, I've stumbled upon this issue because I'm trying to deploy a web application that relies on a properly configured `user.home` property), this property should either be set to `/home/tomcat` or the configured home folder.

That behaviour might be a consequence of this issue (depending on the way that Java or Tomcat determines the default value for this system property) or a separate issue that requires passing an additional command line parameter (i.e. `-Duser.home=/home/tomcat`) to be solved.

**Comment by rafariossaa** [comment]
Hi,
  Thanks for noticing this.
  Do you think if I add `-Duser.home=$TOMCAT_HOME` to the JAVA_OPTS it will do for you ?

**Comment by toKrause** [comment]
That would certainly be an improvement.

In this case, the readme file should reflect the fact that
1. by default, no home folder is available (i.e. remove the reference to the non-existing folder `/home/tomcat`), and
2. that setting `TOMCAT_HOME` alone is not enough. The folder referenced by this environment variable also need to be created and writeability needs to be ensured (i.e. by mounting the referenced folder as a volume).

**Comment by rafariossaa** [comment]
Hi,
  Thanks for your suggestions.
  In fact, `TOMCAT_HOME` is currently set to `/opt/bitnami/tomcat` that is a directory that exists and the user has permissions to  write into it. I am updating the REAMDE too.

**Comment by FraPazGal** [comment]
Hi @toKrause,

From what I can see, the issue should have been resolved with the changes added in `8.5.58-debian-10-r3` version of the image, could you please confirm it so that this issue can be closed? Thanks in advance.
